core/store/migrate/migrations/0162_add_cosmos.sql

```diff
@@ -0,0 +1,52 @@
+-- +goose Up
+
+-- Table Definition ----------------------------------------------
+
+CREATE TABLE cosmos_chains (
+    id text PRIMARY KEY,
+    cfg jsonb NOT NULL DEFAULT '{}'::jsonb,
+    created_at timestamp with time zone NOT NULL,
+    updated_at timestamp with time zone NOT NULL,
+    enabled boolean NOT NULL DEFAULT true
+);
+
+-- Table Definition ----------------------------------------------
+
+CREATE TABLE cosmos_msgs (
+    id BIGSERIAL PRIMARY KEY,
+    cosmos_chain_id text NOT NULL REFERENCES cosmos_chains(id) ON DELETE CASCADE,
```

Comment:
If we drop this reference, can we avoid creating the chains and nodes tables in the first place?
Otherwise they will just be deleted soon with https://smartcontract-it.atlassian.net/browse/BCF-1474